### ISSUE TITLE
checkpoint path

### ISSUE THREAD

**Issue by YoungJ-Baek** [NVIDIA/flownet2-pytorch#199]
after I command 
`python main.py --inference --model FlowNet2 --save_flow --inference_dataset MpiSintelClean \
> --inference_dataset_root /path/to/mpi-sintel/clean/dataset \
> --resume /path/to/checkpoints `

I get 
`Parsing Arguments
  [0.020s] batch_size: 8
  [0.020s] crop_size: [256, 256]
  [0.020s] fp16: False
  [0.020s] fp16_scale: 1024.0
  [0.020s] gradient_clip: None
  [0.020s] inference: True
  [0.020s] inference_batch_size: 1
  [0.020s] inference_dataset: MpiSintelClean
  [0.020s] inference_dataset_replicates: 1
  [0.020s] inference_dataset_root: /path/to/mpi-sintel/clean/dataset
  [0.020s] inference_n_batches: -1
  [0.020s] inference_size: [-1, -1]
  [0.020s] inference_visualize: False
  [0.020s] log_frequency: 1
  [0.020s] loss: L1Loss
  [0.020s] model: FlowNet2
  [0.020s] model_batchNorm: False
  [0.020s] model_div_flow: 20.0
  [0.020s] name: run
  [0.020s] no_cuda: False
  [0.020s] number_gpus: 1
  [0.020s] number_workers: 8
  [0.020s] optimizer: Adam
  [0.020s] optimizer_amsgrad: False
  [0.020s] optimizer_betas: (0.9, 0.999)
  [0.020s] optimizer_eps: 1e-08
  [0.021s] optimizer_lr: 0.001
  [0.021s] optimizer_weight_decay: 0
  [0.021s] render_validation: False
  [0.021s] resume: /path/to/checkpoints
  [0.021s] rgb_max: 255.0
  [0.021s] save: ./work
  [0.021s] save_flow: True
  [0.021s] schedule_lr_fraction: 10
  [0.021s] schedule_lr_frequency: 0
  [0.021s] seed: 1
  [0.021s] skip_training: False
  [0.021s] skip_validation: False
  [0.021s] start_epoch: 1
  [0.021s] total_epochs: 10000
  [0.021s] train_n_batches: -1
  [0.021s] training_dataset: MpiSintelFinal
  [0.021s] training_dataset_replicates: 1
  [0.021s] training_dataset_root: ./MPI-Sintel/flow/training
  [0.021s] validation_dataset: MpiSintelClean
  [0.021s] validation_dataset_replicates: 1
  [0.021s] validation_dataset_root: ./MPI-Sintel/flow/training
  [0.021s] validation_frequency: 5
  [0.021s] validation_n_batches: -1
  [0.023s] Operation finished

Source Code
  Current Git Hash: b'21cd424e05b52c71c2079e405575a32a0726c92f'

Initializing Datasets
  [0.000s] Operation finished

Building FlowNet2 model
  [2.041s] Effective Batch Size: 8
  [2.042s] Number of parameters: 162518834
  [2.042s] Initializing CUDA
  [4.549s] Parallelizing
  [4.550s] nt founNo checkpoid at '/path/to/checkpoints'
  [4.550s] Operation failed
`

How can I solve `nt founNo checkpoid at '/path/to/checkpoints'` problem?
Do I have to make checkpoints folder in flownet2-pytorch folder?
Also, I want to know is there anything I should do except things in readme

**Comment by GUWOGANSHOU** [comment]
You should download the checkpoints and put it in the right location.

**Comment by GUWOGANSHOU** [comment]
May be this blog will be helpful.https://cloud.tencent.com/developer/article/1461933
